controllers/endpoints-controller.js

```diff
@@ -0,0 +1,13 @@
+const fs = require("fs/promises");
+
+exports.getEndpoints = (request, response, next) => {
+  return fs
+    .readFile(`./endpoints.json`)
+    .then((endPoints) => {
+      const parsedEndpoints = JSON.parse(endPoints);
+      return parsedEndpoints;
+    }).then((parsedEndpoints) => {
+      response.status(200).send({ parsedEndpoints });
+    })
```

Comment:
I would recommend a mild refactor here. Have a think, you've required in fs to read a json file to a string, you've then parsed that string right back to a json file and sent it back to the client... What could you require in that matches the end goal of a json object?

Reply:
I'm not sure. The only thing I can think of is express.json